shared/tracker/header.render.desktop.js

```diff
@@ -106,7 +106,8 @@ const styles2 = {
     top: 0,
     ...globalStyles.flexBoxRow,
     height: 90,
-    width: 320
+    width: 320,
+    transition: 'background 0.2s linear'
```

Comment:
maybe we want to move this into style-guide along w/ the other transition (we could also just try it)